contracts/series/SeriesController.sol

```diff
@@ -177,19 +177,19 @@ contract SeriesController is
             if (settlementPrice >= currentSeries.strikePrice) {
                 // OTM
                 writerShare = getCollateralPerOptionToken(
-                    _seriesId,
+                    currentSeries,
```

Comment:
SeriesController.sol and  ISeriesController.sol  should use the same naming as AmmDataProvider.sol for parameters: currentSeries -> series

Reply:
yes lets make these both series